pennylane/qchem/vibrational/taylor_ham.py

```diff
@@ -0,0 +1,704 @@
+
+    **Example**
+
+    >>> one_mode, two_mode = [np.array([[-0.00088528, -0.00361425,  0.00068143]]), np.array([[[0., 0., 0., 0., 0., 0.]]])]
+    >>> freqs=np.array([0.025])
+    >>> uloc=np.array([[1.0]])
+    >>> qml.qchem.taylor_bosonic(coeffs=[one_mode, two_mode], freqs=freqs, uloc=uloc)
+        BoseSentence(
+            {
+                BoseWord({(0, 0): "+", (1, 0): "+", (2, 0): "+"}): -0.0012778303419517393,
+                BoseWord({(0, 0): "+", (1, 0): "+", (2, 0): "-"}): -0.0038334910258552178,
+                BoseWord({(0, 0): "+"}): -0.0038334910258552178,
+                BoseWord({(0, 0): "+", (1, 0): "-", (2, 0): "-"}): -0.0038334910258552178,
+                BoseWord({(0, 0): "-"}): -0.0038334910258552178,
+                BoseWord({(0, 0): "-", (1, 0): "-", (2, 0): "-"}): -0.0012778303419517393,
+                BoseWord({(0, 0): "+", (1, 0): "+"}): (0.0005795050000000001 + 0j),
+                BoseWord({(0, 0): "+", (1, 0): "-"}): (0.026159009999999996 + 0j),
+                BoseWord({}): (0.012568432499999997 + 0j),
+                BoseWord({(0, 0): "-", (1, 0): "-"}): (0.0005795050000000001 + 0j),
+                BoseWord(
+                    {(0, 0): "+", (1, 0): "+", (2, 0): "+", (3, 0): "+"}
+                ): 0.00017035749999999995,
+                BoseWord(
+                    {(0, 0): "+", (1, 0): "+", (2, 0): "+", (3, 0): "-"}
+                ): 0.0006814299999999998,
+                BoseWord(
+                    {(0, 0): "+", (1, 0): "+", (2, 0): "-", (3, 0): "-"}
+                ): 0.0010221449999999997,
+                BoseWord(
+                    {(0, 0): "+", (1, 0): "-", (2, 0): "-", (3, 0): "-"}
+                ): 0.0006814299999999998,
+                BoseWord(
+                    {(0, 0): "-", (1, 0): "-", (2, 0): "-", (3, 0): "-"}
+                ): 0.00017035749999999995,
+            }
+        )
+    """
+    if is_local:
+        start_deg = 2
+    else:
+        start_deg = 3
+
+    harm_pot = _taylor_harmonic(coeffs, freqs)
+    ham = _taylor_anharmonic(coeffs, start_deg) + harm_pot
+    kin_ham = _taylor_kinetic(coeffs, freqs, is_local, uloc)
+    ham += kin_ham
+    return ham.normal_order()
+
+
+def taylor_hamiltonian(
+    pes, max_deg=4, min_deg=3, mapping="binary", n_states=2, ps=False, wire_map=None, tol=None
+):
+    """Return Taylor vibrational Hamiltonian.
+
+    Args:
+        pes (VibrationalPES): object containing the vibrational potential energy surface data
+        max_deg (int): maximum degree of Taylor form polynomial
+        min_deg (int): minimum degree of Taylor form polynomial
+        mapping (str): Mapping used to map to qubit basis. Input values can be ``"binary"`` or ``"unary"``.
+            Default is `"binary"`.
+        n_states(int): maximum number of allowed bosonic states
+        ps (bool): Flag to return the result as a PauliSentence instead of an
+            operator. Defaults to ``False``.
```

Comment:
I think we can remove this and make it False, just to be consistent with molecular Hamiltonian